linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_tls/ansible/shared.yml

```diff
@@ -5,21 +5,27 @@
 # disruption = low
 {{{ ansible_instantiate_variables("rsyslog_remote_loghost_address") }}}
 
-- name: "Get omfwd configuration directive"
-  shell: sed -e '/^action\s*(\s*type\s*=\s*"omfwd"/,/)/!d' /etc/rsyslog.conf /etc/rsyslog.d/*.conf || true
-  register: include_omfwd_config_output
+- name: "{{{ rule_title }}}: Find all files matching /etc/rsyslog.d/*.conf"
+  find:
```

Comment:
Could you use the complete name for the module, please?
In this case: `ansible.builtin.find`